src/Comparison/IfBetweenHelper.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ * This file is part of Handlebars.php Helpers Set
+ *
+ * (c) Dmitriy Simushev <simushevds@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JustBlackBird\HandlebarsHelpers\Comparison;
+
+use Handlebars\Context;
+use Handlebars\Helper as HelperInterface;
+use Handlebars\Template;
+
+/**
+ * Conditional helper that checks if two values are equal or not.
+ *
+ * Example of usage:
+ * ```handlebars
+ *   {{#ifBetween variable min max}}
+ *     The first argument is between min and max
+ *   {{else}}
+ *     The first argument is not between min and max
+ *   {{/ifBetween}}
+ * ```
+ *
+ * @author Jesse Weigert <jesse.weigert@accretivetg.com>
+ */
+class IfBetweenHelper implements HelperInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function execute(Template $template, Context $context, $args, $source)
+    {
+        $parsed_args = $template->parseArguments($args);
+        if (count($parsed_args) != 3) {
+            throw new \InvalidArgumentException(
+                '"ifBetween" helper expects exactly three arguments.'
+            );
+        }
+
+        $variable = $context->get($parsed_args[0]);
+        $min = $context->get($parsed_args[1]);
+        $max = $context->get($parsed_args[2]);
+
+        $condition = ($variable >= $min && $variable < $max);
```

Comment:
Well, if we're talking about your use case I don't think that providing a discount for an order with zero total make any sense. So it should be like:

```handlebars
{#ifEqual total 0}You should order something to get a discount{/ifEqual}
{#ifBetween total 0 100}$10 discount{/ifBetween}
{#ifBetween total 100 200}$15 discount{/ifBetween}
```

In such case `ifBetween` helper should be treated as a strict inequality (`min < value < max`).